src/deluge/gui/views/view.cpp

```diff
@@ -993,7 +993,7 @@ void View::setModLedStates() {
 #if DELUGE_MODEL != DELUGE_MODEL_40_PAD
 	bool affectEntire = getRootUI() && getRootUI()->getAffectEntire();
 	if (!itsTheSong) {
-		if (getRootUI() != &instrumentClipView) affectEntire = true;
+		if (getRootUI() != &instrumentClipView && getRootUI() != &keyboardScreen) affectEntire = true;
```

Comment:
won't this change behavior for non-kit clips as well?

Reply:
Yeah worth checking, I didn't notice this in the original commit

Reply:
No - this just enables turning off affect entire while in the drum keyboard screen. In synth/midi/cv clip's keyboard view the underlying clip always has affect entire on so there's no change in behaviour